site/docs/expressions/extended_expression.md

```diff
@@ -0,0 +1,19 @@
+# Extended expression
```

Comment:
This prose can be cleaned up in a future PR but it isn't very clear to me.  If you ping me with a reminder if this merges then I will be happy to make an attempt at doing so.

Reply:
I tried finding out a suitable name but seems it doesn't quite work. My user scenario is expression level evaluation offloading but current `Expression` doesn't contain all necessary infos required, like schema, function, etc.

Reply:
Sorry, I meant the entire file.  Just minor grammatical things.  It's not a big concern.